### ISSUE TITLE
MG3.8 - can't initialise audio on Android 6.0 and 7.1

### ISSUE THREAD

**Issue by shimfish** [MonoGame/MonoGame#7335]
Creating a SoundEffect object produces an exception on Android 6.0. and 7.1.2 devices, including Amazon Fire HD 8 2018

It seems to produce one of two possible errors:

>Microsoft.Xna.Framework.Audio.NoAudioHardwareException: Audio has failed to initialize. Call SoundEffect.Initialize() before sound operation to get more specific errors.
  at Microsoft.Xna.Framework.Audio.SoundEffect..ctor (System.Byte[] buffer, System.Int32 offset, System.Int32 count, System.Int32 sampleRate, Microsoft.Xna.Framework.Audio.AudioChannels channels, System.Int32 loopStart, System.Int32 loopLength) [0x00028] in <f4b5bf3a10334b9ab5945e18622c0769>:0
  at Microsoft.Xna.Framework.Audio.SoundEffect..ctor (System.Byte[] buffer, System.Int32 sampleRate, Microsoft.Xna.Framework.Audio.AudioChannels channels) [0x00000] in <f4b5bf3a10334b9ab5945e18622c0769>:0

or

>System.DllNotFoundException: Couldn't initialize OpenAL because the native binaries couldn't be found.
  at Microsoft.Xna.Framework.Audio.OpenALSoundController..ctor () [0x00022] in <f4b5bf3a10334b9ab5945e18622c0769>:0
  at Microsoft.Xna.Framework.Audio.OpenALSoundController.EnsureInitialized () [0x00014] in <f4b5bf3a10334b9ab5945e18622c0769>:0
  at Microsoft.Xna.Framework.Audio.SoundEffect.PlatformInitialize () [0x00000] in <f4b5bf3a10334b9ab5945e18622c0769>:0
  at Microsoft.Xna.Framework.Audio.SoundEffect.Initialize () [0x0001d] in <f4b5bf3a10334b9ab5945e18622c0769>:0
  at Microsoft.Xna.Framework.Audio.SoundEffect..ctor (System.Byte[] buffer, System.Int32 offset, System.Int32 count, System.Int32 sampleRate, Microsoft.Xna.Framework.Audio.AudioChannels channels, System.Int32 loopStart, System.Int32 loopLength) [0x00011] in <f4b5bf3a10334b9ab5945e18622c0769>:0
  at Microsoft.Xna.Framework.Audio.SoundEffect..ctor (System.Byte[] buffer, System.Int32 sampleRate, Microsoft.Xna.Framework.Audio.AudioChannels channels) [0x00000] in <f4b5bf3a10334b9ab5945e18622c0769>:0

#### What version of MonoGame does the bug occur on:
- MonoGame 3.8 release nuget android

#### What operating system are you using:
- MacOS

#### What MonoGame platform are you using:

- Android

**Comment by NicusorN5** [comment]
Does it work on 3.7?

**Comment by shimfish** [comment]
I can’t confirm as I actually don’t possess one of the affected devices. This is from live crash reports. 

Certainly worked on 3.6.

**Comment by VictorRabotko** [comment]
3.6 was the last error-free version.
Since 3.7 version I always getting random crushes on random devices. And it is always a "openal audio" problem

**Comment by harry-cpp** [comment]
Duplicate of #6738 I suppose.

**Comment by harry-cpp** [comment]
I'm gonna keep this one open and close the other one.

**Comment by shimfish** [comment]
This doesn't crash on startup like the other bug but only when trying to instantiate a SoundEffect object. Maybe that's just because 3.8 defers the loading of the openal lib?

**Comment by Jjagg** [comment]
Yes, the audio system has been changed so it does not crash on startup so users can use other audio solutions (like fmod).

The underlying issue is the same.

**Comment by SimonDarksideJ** [comment]
Are there any docs yet for swapping the audio backend, say for fmod, available @Jjagg as a possible workaround?

**Comment by Apostolique** [comment]
According to #6629 the audio is already optional.
